src/wizard/java/spring.md

```diff
@@ -0,0 +1,85 @@
+---
+name: Spring
+doc_link: https://https://docs.sentry.io/platforms/java/guides/spring/
+support_level: production
+type: framework
+---
+
+<Alert level="info">
+    Sentry's integration with Spring supports Spring Framework 5.1.2 and above to report unhandled exceptions and optional user information. If you're on an older version, use <a href=https://docs.sentry.io/platforms/java/guides/spring/legacy/>our legacy integration</a>.
+</Alert>
+
+Install Sentry's integration with Spring via Maven, Gradle, or SBT:
+
+```xml {tabTitle:Maven}
+<dependency>
+    <groupId>io.sentry</groupId>
+    <artifactId>sentry-spring</artifactId>
+    <version>{{ packages.version('sentry.java', '3.2.0') }}</version>
+</dependency>
+```
+
+```groovy {tabTitle:Gradle}
+implementation 'io.sentry:sentry-spring:{{ packages.version('sentry.java', '3.2.0') }}'
+```
+
+```scala {tabTitle: SBT}
+libraryDependencies += "io.sentry" % "sentry-spring" % "{{ packages.version('sentry.java', '3.2.0') }}"
+```
```

Comment:
Since this will render one on top of the other maybe we just keep the two most common (gradle and maven)

```suggestion
```

Reply:
Yes, and we'll add short preceding text since we don't have tabs.